backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresDocumentationUnitRepositoryImpl.java

```diff
@@ -171,7 +172,7 @@ public void saveKeywords(DocumentUnit documentUnit) {
 
                 List<String> keywords = contentRelatedIndexing.keywords();
                 for (int i = 0; i < keywords.size(); i++) {
-                  String value = keywords.get(i);
+                  String value = StringUtils.normalizeSpace(keywords.get(i));
```

Comment:
No I think, that is not possible, because the many to many relations like keywords, fields of law, procedures, are saved before the actual jpa save method.They have their own save methods on repo level. The other normalizations are all in the respective transformers ... there comes no option to my mind to do it all in one point unfortunately

Reply:
But I will ask Ralph, if he knows a way